lighty-modules/lighty-netconf-sb/src/main/java/io/lighty/modules/southbound/netconf/impl/util/NetconfConfigUtils.java

```diff
@@ -201,13 +304,27 @@ public static AAAEncryptionService createAAAEncryptionService(AaaEncryptServiceC
      * @param configuration Netconf southbound configuration where should be services injected.
      * @return Netconf southbound configuration with injected services from Lighty core.
      */
-    private static NetconfConfiguration injectClient(final LightyServices services,
-                                                     final NetconfConfiguration configuration) {
+    @Deprecated
+    private static NetconfConfiguration injectClient(final NetconfConfiguration configuration,
+                                                     final LightyServices services) {
         final NetconfClientDispatcher client =
                 new NetconfClientDispatcherImpl(services.getBossGroup(), services.getWorkerGroup(),
                         services.getTimer());
         configuration.setClientDispatcher(client);
         return configuration;
     }
 
+    /**
+     * Inject netconf client dispatcher to Netconf southbound configuration, it uses {@link LightyServices}.
```

Comment:
It does not use LightyServices. I would also delete this method, it does nothing, user can achieve the same thing with one line.